sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp

```diff
@@ -95,11 +97,12 @@ class DistortionCorrector : public DistortionCorrectorBase
     static_cast<T *>(this)->undistortPointImplementation(
       it_x, it_y, it_z, it_twist, it_imu, time_offset, is_twist_valid, is_imu_valid);
   };
+  void convertMatrixToTransform(const Eigen::Matrix4f & matrix, tf2::Transform & transform);
```

Comment:
Is using a void return type necessary? (in this node there are some optimizations, that get turned off is we return things, but I am not sure about this one)

Reply:
I think I seen how tf2_eigen do this and across libraries it's 50/50.